unitTests/magicbox/ResultPreviewsManagerTest.ts

```diff
@@ -0,0 +1,123 @@
+import { ResultPreviewsManager, ISearchResultPreview } from '../../src/magicbox/ResultPreviewsManager';
+import { IMockEnvironment, MockEnvironmentBuilder } from '../MockEnvironment';
+import { $$ } from '../../src/utils/Dom';
+import { ResultPreviewsManagerEvents, IPopulateSearchResultPreviewsEventArgs } from '../../src/events/ResultPreviewsManagerEvents';
+import { toArray } from 'lodash';
+
+export function ResultPreviewsManagerTest() {
+  describe('ResultPreviewsManager', function() {
+    let env: IMockEnvironment;
+    function buildEnvironment() {
+      return (env = new MockEnvironmentBuilder().build());
+    }
+
+    let container: HTMLElement;
+    function buildContainer() {
+      return buildEnvironment().root.appendChild((container = $$('div').el));
+    }
+
+    let suggestionsListboxElement: HTMLElement;
```

Comment:
I don't think this variable is used anywhere. Could it be removed? It's good to keep the number of `describe`-block-scope variables to a minimum where possible.

Reply:
I'll remove it